pubtools/_quay/signature_handler.py

```diff
@@ -580,10 +580,44 @@ def sign_operator_images(self, iib_results):
         for version, iib_details in sorted(iib_results.items()):
             iib_result = iib_details["iib_result"]
             signing_keys = iib_details["signing_keys"]
+            # Version acts as a tag of the index image
             claim_messages += self.construct_index_image_claim_messages(
                 iib_result.index_image_resolved, version, signing_keys
             )
-        LOG.info("claim messages: {0}".format(json.dumps(claim_messages)))
+
+        signature_messages = self.get_signatures_from_radas(claim_messages)
+        self.validate_radas_messages(claim_messages, signature_messages)
+
+        self.upload_signatures_to_pyxis(
+            claim_messages,
+            signature_messages,
+            self.target_settings.get(
+                "sigstore_max_upload_items", self.DEFAULT_MAX_ITEMS_PER_UPLOAD_BATCH
+            ),
+        )
+
+    def sign_task_index_image(self, build_details, signing_key, index_image, tag=None):
```

Comment:
Seems parameter 'index_image' is unnecessary if both index_image and tag can be got from build_details. And build_details is unused in this function.

Reply:
You're right, but if the index image name has to be changed to "iib", it would probably be preferable to keep the "index_image" argument and remove "build_details". I'll wait for confirmation from @midnightercz

Reply:
Removed the redundant parameter